_sections/home-english.md

```diff
@@ -148,7 +148,19 @@ We can also add multiple choice questions that allow multiple answers to be sele
 | ============   | ============= | ========================= |
 | choices        |               |                           |
 
-### Specify other
+#### Choice names
+
+The `name` column of the choices sheet defines the values that will be saved when each choice is selected during data collection. Choice names for **select_multiple** questions are not allowed to contain spaces because when multiple answers are selected, those are separated by spaces. Choice names for **select_one** questions may contain spaces but avoiding them may make analysis easier and leaves the door open for converting the question to a **select_multiple**.
```

Comment:
The select_multiple choices language reads weird to me. What about this:
> Choice names for **select_multiple** questions must not contain spaces because spaces are used as a separator when the answer is saved. 

"leaves the door open" might be a bit too colloquial. What about 
> and leaves the door open for converting the question to a **select_multiple**  at a later date.

Reply:
Thanks!